Makefile

```
@@ -0,0 +1,12 @@
+SRC_DIR=src
+TEST_DIR=tests
+
+install-dev:
+	@pip install -r requirements-dev.txt
+
+format:
+	@black ${SRC_DIR} ${TEST_DIR}
+	@isort ${SRC_DIR} ${TEST_DIR}
+
+test:
+	@python -m pytest
```

If you have a single test folder I'd put it explicitly, especially if you had it included in `$TEST_DIR`

---

Do you mean like this?
```suggestion
	@python -m pytest ${TEST_DIR}
```